resources/cdk/cognito_scenario_user_pool_with_mfa/README.md

```diff
@@ -0,0 +1,118 @@
+# Cognito user pool with MFA scenario resources
+
+## Overview
+
+Creates AWS resources for Amazon Cognito scenarios. The scripts in this
+example create the following resources:
+ 
+* An Amazon Cognito user pool that is configured to:
+    * allow self sign-up
+    * verify by email
+    * require multi-factor authentication (MFA)
+    * allow device tracking
+* An associated client application that can authenticate with the user pool. 
```

Comment:
Remove ending punctuation (for parity with other list items / because this is part of a bulleted list)

Reply:
fixed